packages/cloud_firestore/ios/Classes/CloudFirestorePlugin.m

```diff
@@ -67,22 +68,98 @@
     NSArray *startAtValues = startAt;
     query = [query queryStartingAtValues:startAtValues];
   }
+  id startAtDocument = parameters[@"startAtDocument"];
+  if (startAtDocument) {
+    NSArray *startAtValues = startAtDocument;
+    NSMutableArray *startAtQueryValues = [[NSMutableArray alloc] init];
+    query = [query queryOrderedByFieldPath:FIRFieldPath.documentID descending:NO];
```

Comment:
Since we are at it now, should we expose `descending` to dart as well?

Reply:
It's a reasonable question, but I don't think this has have a corresponding equivalent in the Android or iOS API, so I don't think we need to worry about it.